test/dlc_tests/ec2/test_smclarify.py

```diff
@@ -16,7 +16,8 @@
 @pytest.mark.integration("smclarify_cpu")
 @pytest.mark.model("N/A")
 @pytest.mark.parametrize("ec2_instance_type", SMCLARIFY_EC2_CPU_INSTANCE_TYPE, indirect=True)
-def test_smclarify_metrics_cpu(training, ec2_connection, region, ec2_instance_type, cpu_only, py3_only):
+@pytest.mark.skip(reason="Skipping for patch branch for MX 1.7 CPU")
+def test_smclarify_metrics_cpu(training, ec2_connection, region, ec2_instance_type, cpu_only, py3_only, tf23_and_above_only, mx18_and_above_only):
     run_smclarify_bias_metrics(training, ec2_connection, region, ec2_instance_type)
 
 
```

Comment:
Need to give `tf23_and_above_only, mx18_and_above_only` as input to GPU test as well

Reply:
test_smclarify_metrics_gpu works for mxnet 1.7

Reply:
It skips because mxnet 1.7 is not cu110